.github/workflows/386-environment/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Setup golang 386, Setup Environment"
+description: "Setup golang 386, Set Environment Variables"
+
+# NOTE
+# x86 binary compile environment needed golang@386, mingw32.
+# why "Setup golang x86" : setup-go can not specify "arch"
+
+runs:
+  using: "composite"
+  steps:
+  - name: Set PATH for MINGW32
+    shell: pwsh
+    run: |
+      echo "$env:RUNNER_TEMP/msys/msys64/mingw32/bin/" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+  - name: Setup golang x86
+    shell: pwsh
+    run: |
+      $ProgressPreference = 'SilentlyContinue'
+      Invoke-WebRequest -Uri https://golang.org/dl/go1.14.13.windows-386.zip -OutFile ~/go.zip
```

Comment:
Maybe it's useful if this action takes go version as parameter?

Reply:
Indeed, I added parameter support at https://github.com/mackerelio/mackerel-agent/pull/684/commits/ed5877a7032cf9e58e38c3bb27b678b5f6ab1ca5